lib/msal-core/src/error/ClientConfigurationError.ts

```diff
@@ -169,4 +173,9 @@ export class ClientConfigurationError extends ClientAuthError {
 
         return new ClientConfigurationError(code, `${desc} mising values: ${missingKeys.join(",")}`);
     }
+
+    static createBlockTokenRequestsInHiddenIframeError(): ClientConfigurationError {
```

Comment:
This should probably be a ClientAuthError, not a ClientConfig error.

Reply:
Agreed and modified.